Dockerfile

```diff
@@ -1,13 +1,12 @@
 FROM glassfish
 MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
 
-RUN apt-get update
+WORKDIR /home/OpenDF
 
-RUN mkdir -p /home/OpenDF
-RUN apt-get install git
-RUN git clone https://github.com/scorelab/OpenDF.git /home/OpenDF
-RUN chmod 755 /home/OpenDF/scripts/setup.sh
-RUN /home/OpenDF/scripts/setup.sh
+COPY /scripts/setup.sh /scripts/setup.sh
```

Comment:
Actually, OpenDF is a bit complex project. It needs a lot of files in the project directory to build. For the moment, let's add all the files.

You can use `COPY . .` here.